linter/rules/include_should_not_have_views_with_wildcard.py

```diff
@@ -0,0 +1,15 @@
+from linter.rule import Rule
+from typing import Any, Tuple
+from re import search
+
+
+class IncludeShouldNotHaveViewsWithWildcard(Rule):
+    def applies_to() -> Tuple[str, ...]:
+        return ('include',)
+
+    def run(self, include: Any) -> bool:
+        pattern = r'\*\.view'
+        wildcard_include = [bool(search(pattern, i)) for i in include]
```

Comment:
`include` would just contain a single string, or, in the case of the suggestion above, a single dict with { "name": i }...so there's no need to loop through include here, as it's just one thing.